devicon.json

```
@@ -6654,6 +6654,27 @@
         "color": "#dd4814",
         "aliases": []
     },
+    {
+        "name": "unifiedmodelinglanguage",
+        "altnames": [
+            "uml"
```

Add name, but with spaces
```suggestion
            "uml",
            "unified modeling language"
```